tests/backup/backup_test.go

```diff
@@ -2777,7 +2781,12 @@ var _ = Describe("{ClusterBackupShareToggle}", func() {
 
 				userBackups, err := GetAllBackupsForUser(username, password)
 				log.FailOnError(err, "Fail on Fetching backups for %s", username)
-
+				if len(userBackups) == 0 {
+					time.Sleep(2 * time.Minute)
```

Comment:
We need not use `time.Sleep` here. Create a function which gets all the backups for the user and add it to `task.DoRetryWithTimeout`
Set the timeout to 2 min and polling interval to 10 sec.